php/libraries/NDB_BVL_Feedback.class.inc

```diff
@@ -713,6 +713,13 @@ class NDB_BVL_Feedback
             );
         }
 
+        $nextID = $DB->pselect('SELECT FeedbackID from feedback_bvl_thread ORDER BY FeedbackID DESC LIMIT 0, 1', null);
```

Comment:
`SELECT MAX(FeedbackID) +1 FROM feedback_bvl_threa` is, in my opinion, simpler to understand.

Reply:
And to get rid of the next if/else block:
`select IFNULL(MAX(FeedbackID), 0) + 1 as FeedbackID from feedback_bvl_thread;`